docker/packages_license_check_exclude.json

```diff
@@ -39,7 +39,8 @@
             "docker_images": [
                 "xsoar-tools",
                 "xsoar-tools-hf",
-                "demisto-sdk"
+                "demisto-sdk",
+                "slackbolt"
```

Comment:
@glicht I aded the docker image to the exclude license since it [fails](https://app.circleci.com/pipelines/github/demisto/dockerfiles/81488/workflows/291fc4cd-dfb8-4a19-8ee1-42157886f709/jobs/87642) on pylint (since this image is based on the demisto-sdk image)

Reply:
this is going to be used in a prod image now, not sure if we can use the same reason
we should uninstall pylint in the image, especially if it isnt needed for the usecase